config/datanode.example.toml

```diff
@@ -29,16 +29,36 @@ connect_timeout = "1s"
 # `TCP_NODELAY` option for accepted connections, true by default.
 tcp_nodelay = true
 
-# WAL options, see `standalone.example.toml`.
+# WAL options.
+# Uncomments all other wal options except the one you chose.
 [wal]
 # WAL data directory
+provider = "raft-engine"
```

Comment:
Maybe we should use snake_case. @WenyXu Which do you prefer?
https://github.com/GreptimeTeam/greptimedb/issues/2936

Reply:
Yes, I vote for the `snake_case`,  too.

Reply:
Changed to `snake_case`.